### PR TITLE
Update Family class to use Set for storing multiple pets

### PR DESCRIPTION
**Pull request by ElshanAlasgarov**

### CODE REVIEW

File: src/main/java/az/edu/turing/Entities/Family.java [hunk 1]
```diff
@@ -87,12 +85,12 @@ public void setChildren(Human[] children) {
         this.children = children;
     }
 
-    public Pet getPet() {
-        return pet;
+    public Set<Pet> getPet() {
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/Entities/Family.java]: rename method to `getPets`()

File: src/main/java/az/edu/turing/Entities/Family.java [hunk 2]
```diff
@@ -87,12 +85,12 @@ public void setChildren(Human[] children) {
...
+        return pets;
     }
 
-    public void setPet(Pet pet) {
-        this.pet = pet;
+    public void setPet(Set<Pet> pets) {
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/Entities/Family.java]: rename

File: src/main/java/az/edu/turing/Entities/Human.java [hunk 3]
```diff
@@ -29,12 +30,26 @@ public Human(String name, String surname, int year, int iq, String[][] schedule,
     }
 
     public String greetPet() {
-        return "Hello, " + family.getPet().getNickname();
+        Set<Pet> pets = family.getPet();
+        StringBuilder result = new StringBuilder();
+        for (Pet pet : pets) {
+            result.append(pet.getNickname()).append(", ");
+        }
+        if (result.length() > 2) {
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/Entities/Human.java]: why?

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

File: src/main/java/az/edu/turing/Entities/Human.java [hunk 4]
```diff
@@ -29,12 +30,23 @@ public Human(String name, String surname, int year, int iq, String[][] schedule,
     }
 
     public String greetPet() {
-        return "Hello, " + family.getPet().getNickname();
+        Set<Pet> pets = family.getPets();
+        StringBuilder result = new StringBuilder();
+        for (Pet pet : pets) {
+            result.append(pet.getNickname()).append(", ");
+            result.delete(result.length() - 2, result.length());
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/Entities/Human.java]: why? are you sure this line should be inside loop? Also you can handle this case in another way with if condition

File: src/main/java/az/edu/turing/Entities/Human.java [hunk 5]
```diff
@@ -29,12 +30,23 @@ public Human(String name, String surname, int year, int iq, String[][] schedule,
...
     public String describePet() {
-        String slyLevel = family.getPet().getTrickLevel() > 50 ? "very sly" : "almost not sly";
-        return String.format("I have a %s, it is %d years old, and it is %s.", family.getPet().getSpecies(), family.getPet().getAge(), slyLevel);
+        StringBuilder result = new StringBuilder();
+        for (Pet pet : family.getPets()) {
+            String slyLevel = pet.getTrickLevel() > 50 ? "very sly" : "almost not sly";
+            result.append(String.format("I have a %s, it is %d years old, and it is %s.\n",
+                    pet.getSpecies(), pet.getAge(), slyLevel));
+        }
+        return result.toString().trim();
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/Entities/Human.java]: do we need `trim `here?

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

File: src/main/java/az/edu/turing/Entities/Human.java (hunk 4 above)
Comment (adiliso) [reply src/main/java/az/edu/turing/Entities/Human.java]: We want to greet all pets.

**Pull request merged**